documentation/examples.md

```diff
@@ -0,0 +1,14 @@
+# node-newrelic examples
+
+We have moved our examples and tutorials over to [newrelic-node-examples](https://github.com/newrelic/newrelic-node-examples) as more robust applications. Here are some quick links for implementing custom instrumentation and using our shim API:
```

Comment:
It looks like to get just this as a tutorial you'll need it in its own folder. I would move this to `examples/instrumentat.md` and then add `jsconf.json` like this

```
{
  "instrumentation": {
    "title": "Instrumentation Examples"
  }
}
```

Reply:
Oops just saw this I'll redo what I did